onnxruntime/core/providers/rocm/tensor/transpose_impl.cu

```diff
@@ -4,9 +4,204 @@
 #include "core/providers/rocm/cu_inc/common.cuh"
 #include "core/providers/rocm/tensor/transpose_impl.h"
 
+#define HCC_ENABLE_PRINTF
```

Comment:
(nit):  is this debug code - can it be deleted?

Reply:
Oops, yeah.